alws/crud/test_repository.py

```diff
@@ -139,9 +168,61 @@ async def bulk_create_package_mapping(
     await session.flush()
 
 
+def get_team_role_name(team_name: str, role_name: str):
+    return f'{team_name}_{role_name}'
+
+async def create_test_repository_role_mapping(session: AsyncSession, team_name: str):
+    # print('*MappingDB:0')
+    required_roles = (Contributor, Observer)
```

Comment:
I'd say we should maybe restrict to only team >= Contributor - Given that the Observers don't have the ability to perform any write action